components/atom/tag/src/index.scss

```diff
@@ -2,15 +2,17 @@
 
 $bgc-atom-tag: color-variation($c-gray, 3) !default;
 $c-atom-tag-actionable: $c-white !default;
-$h-atom-tag: $p-xxl !default;
+$h-atom-tag-medium: 32px !default;
```

Comment:
I'm not sure if it should be the case here but for atom-button we expressed sizes with paddings. https://github.com/SUI-Components/sui-components/blob/master/components/atom/button/src/index.scss#L14

The question underneath is "if the paddings grow, should tags vertical padding grow as well?". I'd say  "yes". Sizes of tags and buttons are used to mark space between text and border: a padding. So the height should be expressed in paddings and the text vertically centered. It's my opinion.

Reply:
I agree with you, I'm gonna talk to the UI guys

Reply:
After the conversation the agreement is to use the height of the component